From 6d12b8ae435f0c39578096543003338e6280e75e Mon Sep 17 00:00:00 2001
From: Thomas Gleixner <tglx@linutronix.de>
Date: Sat, 25 Jul 2009 23:40:31 +0200
Subject: [PATCH] affs: use semaphore_init instead of init_MUTEX

commit ca17ebb87927dd486e926b386ff52a5c6d555504 in tip.

The usage of these "mutex" is non obvious and probably a completion
in some places. Make it them semaphores.

Signed-off-by: Thomas Gleixner <tglx@linutronix.de>
Cc: Al Viro <viro@zeniv.linux.org.uk>
Signed-off-by: Paul Gortmaker <paul.gortmaker@windriver.com>
---
 fs/affs/super.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/affs/super.c b/fs/affs/super.c
index 16a3e47..51e326f 100644
--- a/fs/affs/super.c
+++ b/fs/affs/super.c
@@ -114,8 +114,8 @@ static void init_once(void *foo)
 {
 	struct affs_inode_info *ei = (struct affs_inode_info *) foo;
 
-	init_MUTEX(&ei->i_link_lock);
-	init_MUTEX(&ei->i_ext_lock);
+	semaphore_init(&ei->i_link_lock);
+	semaphore_init(&ei->i_ext_lock);
 	inode_init_once(&ei->vfs_inode);
 }
 
-- 
1.7.0.4

